### PR TITLE
 rbg multiplier  while entertainment mode

### PR DESCRIPTION
**Pull request by CaptainRoot**
Add option to enable rgb multiplier while using entertainment mode for white correction

### CODE REVIEW

File: Arduino/Generic_WS2812_Strip/Generic_WS2812_Strip.ino [hunk 1]
```diff
@@ -898,11 +913,28 @@ void entertainment() {
...
+        // Apply multiplier for white correction
+        lights[packetBuffer[i * 4]].currentColors[0] = packetBuffer[i * 4 + 1] * rgb_multiplier[0] / 100;
+        lights[packetBuffer[i * 4]].currentColors[1] = packetBuffer[i * 4 + 2] * rgb_multiplier[1] / 100;
+        lights[packetBuffer[i * 4]].currentColors[2] = packetBuffer[i * 4 + 3] * rgb_multiplier[2] / 100;
+      }
+    }
+    else {
+      for (uint8_t i = 0; i < packetSize / 4; i++) {
+
+        // Apply multiplier for white correction
```
Comment (CaptainRoot) [inline Arduino/Generic_WS2812_Strip/Generic_WS2812_Strip.ino:929]: Wrong comment: this lines do not use the rgb multipliers

**Comment by mariusmotea** [comment]
Hi,

Thanks for your support. My only concert here is the extend of the json file in SPIFFS because i know the conversion to json consume lot of RAM. Why did you choose to have different conversion levels than the default ones?

**Comment by CaptainRoot** [comment]
Hi,

i use the original multiplier values. I only add an option switch to enable or disable  calculation in entertainment mode.
This option is added to json config, 

I thought it will be useful to have the option to switch off the correction of because the calculation will tahe some time and so increases the delay.

**Comment by mariusmotea** [comment]
Theoretically it must not be noticied, did you observe an increase in delay? How many virtual lights did you use in your test?

**Comment by CaptainRoot** [comment]
I only use a short led strip with 4 virtual lights and  around 80 leds. You are right, i did not see any delay here.

Should i remove option to activate or deactivate?

**Comment by mariusmotea** [comment]
I think is safe to use multiplier active also in entarteinment with no enable/disable switch.

**Comment by CaptainRoot** [comment]
Ok, i will remove the option and open a new request

**Pull request closed without merging**